base/docs/helpdb.jl

```diff
@@ -2587,13 +2587,6 @@ Right division operator: multiplication of `x` by the inverse of `y` on the righ
 Base.(:(/))
 
 doc"""
-    ldltfact(A) -> LDLtFactorization
-
-Compute a factorization of a positive definite matrix `A` such that `A=L*Diagonal(d)*L'` where `L` is a unit lower triangular matrix and `d` is a vector with non-negative elements.
```

Comment:
Why are you removing this? Isn't there an `ldltfact` for dense matrices? The docstring that you moved inline was specifically for `SymTridiagonal`.